src/components/ButtonPanel.js

```diff
@@ -68,12 +69,32 @@ const styleButton = {
   justifyContent: 'center',
   alignItems: 'center',
 };
+const rightButtons = {
+  display: 'inline-flex',
+  width: 147,
+  height: 90,
+  border: '1px solid black',
+  justifyContent: 'center',
+  alignItems: 'center',
+  backgroundColor: '#FF8645',
+};
+
+const styleZero = {
+  display: 'inline-flex',
+  width: 296,
+  height: 90,
+  border: '1px solid black',
+  justifyContent: 'center',
+  alignItems: 'center',
+};
+
+
 const Group1 = () => (
   <div style={styleGroup1}>
```

Comment:
You completely  did not get the instructions of this component.
There's a reason you had a `Button` component, it's so that you'll use it here to render the buttons in a "Panel".

The `<div>` talked about in the requirements, is so that you group the buttons, something like this below:

![Screen Shot 2020-04-22 at 10 06 48 PM](https://user-images.githubusercontent.com/40712459/80033861-5532ef00-84e5-11ea-8b44-e8ffc8446715.png)


Also, be there any styles you want to write, carry them into a CSS file. From my explanation, i guess these styles will go away ;)

HAPPY CODING

Reply:
Ok, I will check it